src/screens/GroupInfo/elements/ModalAction.js

```diff
@@ -6,6 +6,7 @@ import SimpleToast from 'react-native-simple-toast';
 import {ANONYMOUS, SIGNED} from '../../../hooks/core/constant';
 import {COLORS} from '../../../utils/theme';
 import Loading from '../../Loading';
+import {fonts} from '../../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `fonts` has been added. Ensure that the `fonts` object is correctly exported from the `../../../utils/fonts` module and contains the required properties used in this file.

```diff
+ import {fonts} from '../../../utils/fonts';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->